frontend/modules/resources/resources.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from "react";
+import "./index.css";
+import ClubResources, { IResourcePage } from "./clubResources";
+
+const res: IResourcePage[] = [
+  {
+    name: "Purdue Hackers",
+    workshop: [
+      {
+        title: "Nodejs Workshop",
+        link:
+          "https://purduehackers.gitbook.io/tutorials/workshops/nodejs-workshop"
+      },
+    ],
+    image: "../../static/images/nodejslogo.png",
```

Comment:
Remove the "../../" in the file path for all of these file paths

Reply:
Done!